dev/release/release-config.jsonc

```
@@ -11,16 +11,16 @@
 {
     // Captain information
     // To get your slack username, navigate to Profile -> More -> Account settings -> Username (at the bottom) -> Expand
-    "captainSlackUsername": "dax",
-    "captainGitHubUsername": "daxmc99",
+    "captainSlackUsername": "bolaji",
```

```suggestion
    "captainSlackUsername": "bolaji.olajide",
```

I should have checked slack before making that suggestion. But here's the correct one,😆.